rfq/manager.go

```diff
@@ -577,12 +595,11 @@ func (m *Manager) mainEventLoop() {
 
 		// Handle subsystem errors.
 		case err := <-m.subsystemErrChan:
-			log.Errorf("Manager main event loop received "+
-				"subsystem error: %v", err)
-
 			// Report the subsystem error to the main server.
-			m.cfg.ErrChan <- fmt.Errorf("encountered RFQ "+
-				"subsystem error: %w", err)
+			m.handleError(
```

Comment:
comment above this line is no longer always true/accurate, depends on what `m.handleError` does